src/deprecated/index.ts

```diff
@@ -22,6 +22,7 @@ export type {
   DropdownItemProps,
   DropdownMenuProps
 } from '../Dropdown'
+export {default as InputField} from '../InputField'
```

Comment:
Are we missing FormGroup and ChoiceInputField here?

Reply:
Similarly those components appear to still be in the main src/index.ts bundle

Reply:
> Are we missing FormGroup and ChoiceInputField here?

Sorry, I thought FormGroup was handled in a previous PR. Not sure what happened to it 🤔 ChoiceInputField can't really be deprecated until ChocieFieldset is deprecated.

Reply:
@pksjce removed it from her one as it was going to be part of FormControl deprecation I think. I haven't seen a separate PR for that either. Can we do it here?